content/shared/influxdb3-admin/upgrade.md

```diff
@@ -0,0 +1,356 @@
+{{% tab-content %}}
+```bash
+docker compose down
+docker compose pull
+docker compose up -d
+```
+{{% /tab-content %}}
+{{% tab-content %}}
+```powershell
+# Download the latest Windows binary
+Invoke-WebRequest `
+  -Uri "https://dl.influxdata.com/influxdb/releases/influxdb3-{{< product-key >}}-{{< latest-patch >}}-windows_amd64.zip" `
+  -OutFile "influxdb3-{{< product-key >}}.zip"
+
+# Extract the binary
+Expand-Archive -Path influxdb3-{{< product-key >}}.zip -DestinationPath . -Force
+
+# Stop the service, replace the binary, and start the service
+Stop-Service influxdb3
+Copy-Item -Path "influxdb3.exe" -Destination "C:\Program Files\InfluxData\influxdb3\" -Force
+Start-Service influxdb3
+```
+{{% /tab-content %}}
+{{< /tabs-wrapper >}}
+
+{{% show-in "enterprise" %}}
+
+## Upgrade a multi-node cluster
+
+Upgrade {{% product-name %}} instances to newer versions using rolling upgrades to minimize downtime.
+When upgrading multi-node clusters, you need to understand catalog version constraints and the recommended upgrade order for different node modes.
+
+- [Upgrade a single {{% product-name %}} instance](#upgrade-a-single--product-name--instance)
+- [Upgrade a multi-node cluster](#upgrade-a-multi-node-cluster)
+  - [Before you upgrade](#before-you-upgrade)
+  - [Multi-node upgrade procedure](#multi-node-upgrade-procedure)
+  - [Upgrade constraints](#upgrade-constraints)
+  - [Troubleshooting](#troubleshooting)
+
+### Catalog version compatibility
+
+{{% product-name %}} uses a catalog to track metadata about measurements, tags, and fields.
+Some versions introduce catalog version updates that affect how nodes can interoperate during rolling upgrades.
+
+> [!Important]
+> #### Important version transitions
+> 
+> - **3.4.x**: Introduced a catalog version update that requires all nodes to upgrade before catalog modifications can resume.
+> - **3.2.x to 3.5.x**: Nodes running 3.2.1 can temporarily coexist with nodes running 3.5.0, but catalog modifications are blocked until all nodes complete the upgrade.
+
+During a rolling upgrade across a catalog version boundary, nodes running older versions cannot modify the catalog.
+This affects writes that add new measurements, tags, or fields, but allows writes to existing measurements, tags, and fields.
+
+
+
+### Multi-node upgrade procedure
+
+Follow these steps to upgrade your {{% product-name %}} deployment with minimal downtime.
+
+#### Recommended node upgrade order
+
+The order in which you upgrade nodes affects the availability of catalog modifications during the upgrade.
+Different node modes have different impacts on catalog updates:
+
+- **Ingest nodes**: Primarily update the catalog when accepting writes that add new measurements, tags, or fields via line protocol.
+- **Query nodes**: Can accept API requests that update the catalog (for example, `influxdb3 create table`), but less frequently than ingest nodes.
+- **Compactor nodes**: Rarely modify the catalog during normal operation.
```

Comment:
Is this true? Every time the compactor compacts a file, doesn't it need to update the Parquet file location in the catalog?